packages/bierzo-wallet/src/routes/welcome/components/ExtensionInteraction.tsx

```diff
@@ -0,0 +1,27 @@
+import { JsRpcRequest } from '@iov/core';
+import React from 'react';
+
+export class ExtensionInteraction extends React.Component {
+  componentDidMount(): void {
+    setInterval(async () => {
+      const extensionId = 'dafekhlcpidfaopcimocbcpciholgkkb';
+
+      const request: JsRpcRequest = {
+        id: 1,
+        method: 'getIdentities',
+        params: {
+          reason: 'I would like to know who you are on Ethereum',
+          chainIds: ['ethereum-eip155-5777'],
+        },
+      };
+
+      chrome.runtime.sendMessage(extensionId, request, response => {
```

Comment:
I think it is easier if the dummy web interacts directly with
```
// This in dummy web page
window.dispatchEvent(myCustomEvent)
const resolveCustomEvent = function (data) {
  window.removeEventListener('MY_CUSTOM_ANSWER_ID', resolveCustomEvent); // if needed
  console.log("Beautiful data received!!");
  console.log(data);
}
window.addEventListener('MY_CUSTOM_ANSWER_ID', resolveCustomEvent)
```

And then in the content script (which is the only place which is able to receive it), code what you have done here, using the `chrome` instance for talking to content script.

Reasons:
- Developers are more familiar with general window JS object
- We simplify the dummy webpage removing chrome instance from it.
- More natural??

Reply:
See discussion about communication channels below